testing/hexer/20-backtrace-guards.patch

```diff
@@ -0,0 +1,32 @@
+diff --git a/apps/Utils.cpp b/apps/Utils.cpp
+index 92d810b..9ea3dc7 100644
+--- a/apps/Utils.cpp
++++ b/apps/Utils.cpp
+@@ -44,7 +44,9 @@
+ #include <cxxabi.h>
+ #include <sys/ioctl.h>
+ #include <sys/wait.h>  // WIFEXITED, WEXITSTATUS
++#ifdef __GLIBC__
+ #include <execinfo.h> // backtrace
++#endif
```

Comment:
It would probably be better to have a compile test that checks for `execinfo.h` and set `HAVE_EXECINFO_H` or similar and then do:
````C
#ifdef HAVE_EXECINFO_H
#include <execinfo.h>
#endif
````